graviti/dataset/struct.py

```diff
@@ -55,3 +56,87 @@ def edit(self, title: Optional[str] = None, description: Optional[str] = None) -
 
     def close(self) -> None:
         """Close the draft."""
+
+
+class Commit(AttrsMixin, ReprMixin):
+    """This class defines the structure of a commit.
+
+    Arguments:
+        commit_id: The commit id.
+        parent_commit_id: The parent commit id.
+        title: The commit title.
+        description: The commit description.
+        committer: The commit user.
+        committed_at: The time when the draft is committed.
+
+    """
+
+    _T = TypeVar("_T", bound="Commit")
+
+    _repr_attrs: Tuple[str, ...] = ("parent_commit_id", "title", "committer", "committed_at")
+
+    commit_id: str = attr(key=camel)
+    parent_commit_id: Optional[str] = attr(key=camel)
+    title: str = attr()
+    description: str = attr(default="")
+    committer: str = attr()
+    committed_at: str = attr()
+
+    def __init__(  # pylint: disable=too-many-arguments
+        self,
+        commit_id: str,
+        parent_commit_id: Optional[str],
+        title: str,
+        description: str,
+        committer: str,
+        committed_at: str,
+    ) -> None:
+        self.commit_id = commit_id
+        self.parent_commit_id = parent_commit_id
+        self.title = title
+        self.description = description
+        self.committer = committer
+        self.committed_at = committed_at
+
+    def _repr_head(self) -> str:
+        return f'{self.__class__.__name__}("{self.commit_id}")'
+
+    @classmethod
+    def loads(cls: Type[_T], contents: Dict[str, Any]) -> _T:
```

Comment:
we do not use `loads` and `dumps` in graviti SDK
use `from_xxx`, `to_xxx` instead